packages/web/components/transactions/transaction-row.tsx

```diff
@@ -15,6 +16,8 @@ export type TransactionStatus = "pending" | "success" | "failed";
 type Effect = "swap" | "deposit" | "withdraw";
 
 interface Transaction {
+  selectedTransaction?: FormattedTransaction;
```

Comment:
We could simplify this type by adding an `isSelected` boolean flag instead of requiring the whole object. Thoughts?